src/sql/workbench/contrib/notebook/browser/cellViews/outputArea.component.ts

```diff
@@ -38,7 +38,6 @@ export class OutputAreaComponent extends AngularDisposable implements OnInit {
 			this._register(this.cellModel.onOutputsChanged(e => {
 				if (!(this._changeRef['destroyed'])) {
 					this._changeRef.detectChanges();
-					this._changeRef.detach();
```

Comment:
Is this being removed since you'll have multiple "outputsChanged" events being fired now?

Reply:
Yes, since there are multiple events, theres no point in detaching.
I added this during my last implementation of streaming because I thought it was causing the table to re-render. After some more digging, I realized the root cause was an @Input field was being updated in the gridOutputComponent.